plugins/GSdx/GSSetting.cpp

```diff
@@ -33,8 +33,11 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_FILTER:
 			return "Control the texture bilinear filtering of the emulation.\n\n"
 				"Nearest:\nAlways disable interpolation, rendering will be blocky.\n\n"
-				"PS2:\nUse same mode as the PS2. It is the more accurate option.\n\n"
-				"Forced:\nAlways enable interpolation. Rendering is smoother but it could generate some glitches.";
+				"Bilinear PS2:\nUse same mode as the PS2. It is the more accurate option.\n\n"
+				"Bilinear Forced:\nAlways enable interpolation. Rendering is smoother but it could generate some glitches.\n\n"
+				"Trilinear:\nUse openGL trilinear interpolation when PS2 uses mipmaps.\n\n"
+				"Trilinear Forced Bilinear:\nSame as above but always enable bilinear interpolation.\n\n"
+				"Trilinear Ultra:\nAlways enable full trilinear interpolation. !WARNING SLOW!.\n\n";
```

Comment:
It is important to specify bilinear filtering since it differs from trilinear filtering.